tests/re.test.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+import fetchMock, { enableFetchMocks } from "jest-fetch-mock";
+
+import SDK, { Rule, RulesPage, Schedule } from "../src/sdk";
+
+enableFetchMocks();
+
+const authUrl = "http://localhost";
+const sdk = new SDK({ authUrl });
+
+describe("Rules SDK", () => {
+  const token = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9";
+  const domainId = "a5a4b1a1-ea89-47ed-af2e-5b38e696d6a1";
+  const queryParams = {
+    offset: 0,
+    limit: 10,
+  };
+  const rule: Rule = {
+    id: "a5a4b1a1-ea89-47ed-af2e-5b38e696d6a1",
+    name: "rule 1",
+    schedule: {
+      start_datetime: "2025-02-07T03:14:00.000Z",
+      time: "0001-01-01T07:39:44.000Z",
+      recurring: "none",
+      recurring_period: 0,
+    },
+    logic: {
+      type: 0,
+      value: "this is the logic",
+    },
+    metadata: {
+      key: "value",
+    },
+  };
+
+  const rulesPage: RulesPage = {
+    rules: [rule],
+    total: 1,
+    offset: 0,
+    limit: 10,
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("Create should create a rule", async () => {
+    fetchMock.mockResponseOnce(JSON.stringify(rule));
+
+    const response = await sdk.Rules.create(domainId, rule, token);
+    expect(response).toEqual(rule);
+  });
+
+  test("List should return a list of rules", async () => {
+    fetchMock.mockResponseOnce(JSON.stringify(rulesPage));
+
+    const response = await sdk.Rules.list(domainId, queryParams, token);
+    expect(response).toEqual(rulesPage);
+  });
+
+  test("view should return a rule", async () => {
```

Comment:
```suggestion
  test("View should return a rule", async () => {
```
Do the same for the others